### PR TITLE
[T4A2][F10-B2]Ravi Shwetha

### PR DESCRIPTION
**Pull request by ravishwetha**
Ready for Review

### CODE REVIEW

**Comment by ravishwetha** [comment]
Sorry about the extra commits, I was merging branches for the previous tutorial to my master before every pull request but I reverted those changes (I tried deleting the forked repo then forking again but the commits were saved for some reason?), so the git diff just shows the changes for this tutorial.

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ public Command parseCommand(String userInput) {
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareAdd(String args){
+    private static Command prepareAdd(String args){
```
Comment (KulaniT) [inline src/seedu/addressbook/parser/Parser.java:99]: The purpose here is to understand when you need to have a class level method. So do you think all the methods in Parser class need to be class-level? Think about it.

**Comment by KulaniT** [comment]
Overall good in code quality and using of access modifiers :)

**Comment by KulaniT** [comment]
@ravishwetha : Some comments added. Please close the PR after reading comments

**Pull request closed without merging**